qemu/tests/cfg/virtio_serial_file_transfer.cfg

```diff
@@ -62,20 +62,27 @@
             migration_protocol = "exec"
             migration_exec_cmd_src = "gzip -c > '%s'"
             migration_exec_cmd_dst = "gzip -c -d '%s'"
-        - iommu_enabled:
-            only q35
-            only unix_socket
-            no WinXP WinVista Win7 Win8 Win8.1 Win2000 Win2003
-            no Win2008 Win2008..r2 Win2012 Win2012..r2
-            intel_iommu = yes
-            virtio_dev_iommu_platform = on
-            iommu_eim = on
-            enable_guest_iommu = yes
-            virtio_dev_ats = on
-            machine_type_extra_params = "kernel-irqchip=split"
     variants:
         - unix_socket:
         - tcp_socket:
             chardev_backend_vs1 = tcp_socket
         - pty:
             chardev_backend_vs1 = pty
+    variants:
+        - @iommu_disabled:                                                              
```

Comment:
@MiriamDeng  How about using `@default` ?

Reply:
Thanks, updated it, could you please help to review it again ? @yanan-fu  Thanks.